### DISCUSSION TITLE
Can't access my account

### DISCUSSION THREAD

**Discussion by MigsRS123** [vercel/community#4016] [Account Issue] [answered]
### Question

I'm trying to create an accout with github, but it says that this account is blocked, I tried to use other emails that weren't linked with github, but either says that is blocked or that it can't complete the process.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by MigsRS123** [reply]
Didn't receive such e-mail

**Reply by amyegan** [reply]
If you email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy, that will open a new ticket with the team that handles account issues. 🙏
